audit.json

```diff
@@ -0,0 +1,3 @@
+{
+
```

Comment:
what's this empty file?

Reply:
This audit.json file use to ignore any false positives from ZAP security testing